### ISSUE TITLE
Update Validation Information

### ISSUE THREAD

**Issue by smiths** [smiths/MEASURE#111]
Hello Michelle.  Can you please update the validation information in the rubric template?  The instructions are in the Departmental Guide in Section 10.  The problem is that the course information names in the Vena database do not match the list that Maria created for Evan.  In a few minutes, I'll be uploading a new version of the guide that includes the instructions on finding the specific strings used to represent the courses inside the database.  I'll also e-mail you the file that holds the information.  You can find the relevant information starting around row 194.  A sample problem is that the Vena database uses ENGINEER, but the Maria generated information used ENGN or ENGI etc. 

Please let me know if you have any questions.

**Comment by smiths** [comment]
Hello Michelle.  This would be nice to have working before next week.  Do you think you can get to it?

**Comment by zhengm2** [comment]
Hi @smiths , I am aiming to have this done by the end of this week.

**Comment by smiths** [comment]
Sounds good Michelle.  Please let me know if you have any questions.

**Comment by zhengm2** [comment]
Hi @smiths , I finally have a little bit of time to work on this. Does this need to match up with how the courses are laid out in the program maps? For example, is 'CHEM ENG' fine because that's how the program map is laid out even though it does not match with Mosaic. I'm looking at some previous emails and it seems like the list that was sent to Evan would have listed the courses as "CHEMENG, SFWRENG, etc" as one word without the space.

Thanks,
Michelle

**Comment by zhengm2** [comment]
Hi @smiths , I finished making changes but when I go to save the template I get the following message. Is it safe to choose Yes? Perhaps this is something @EvanVena can help answer?

![capture](https://cloud.githubusercontent.com/assets/20647879/21728132/936b0424-d412-11e6-86fc-a0a2e9384b0d.JPG)

Thank you,
Michelle

**Comment by smiths** [comment]
Hello Michelle.  Yes, it is safe to save the template.  Go ahead and click Yes.

**Comment by zhengm2** [comment]
Thank you Spencer. It has been saved. Please let me know if anything else needs to be done for this.

**Comment by smiths** [comment]
Hello Michelle.  Did you get a chance to test the changes?  That is, when you enter a valid course/term/section combination, does the error message no longer appear?  If you test a few cases successfully, then we can close this issue.  We don't need it to be 100% accurate, since if a faculty member finds a mistake, we now know how to fix it.

**Comment by zhengm2** [comment]
Hi Spencer, I checked out a couple Rubric Input Templates and did not receive any error messages. However, I'm not 100% sure I'm checking in the correct place as I never encountered the message previously because I didn't input any data.

**Comment by smiths** [comment]
Hello Michelle.  I have verified a few of your selections.  The only "mistake" I found was for my course from Term 1 (SFWR ENG 3XA3), but I think that happened because I already fixed my class and then you "fixed" it again.  From what I can tell, the course list for the rubrics looks good.  I'll close this issue.  We can always address any courses where the instructor notices this message coming up in error.
